src/main/java/com/google/sps/scheduler/Scheduler.java

```diff
@@ -37,6 +38,7 @@
   public static final int END_HOUR = 18;
   public static final int END_MINUTE = 0;
   public static final long DEFAULT_DURATION_IN_MILLISECONDS = TimeUnit.MINUTES.toMillis(30);
+  // TODO: remove the sample tasks after the Tasks API integration (issue #8)
   public static final ImmutableList<String> TASKS_SAMPLE = ImmutableList.of(
```

Comment:
Sorry for being late to the party, but can these sample tasks be at least moved to some other file? I imagine you'd like to show them in the UI, and accessing them from Scheduler does not seem right.

Reply:
Yes, it makes sense.